### DISCUSSION TITLE
Upload publish date through REST API

### DISCUSSION THREAD

**Discussion by joao-pedrozo** [pluginpal/strapi-plugin-publisher#27] [Ideas]
I have a list of content with publish dates from the future and I would like to POST them to Strapi through API with the publish date schedule included, this plugin allows me to do this?

**Comment by ComfortablyCoding** [comment]
This is not possible at the moment. It should be possible as an enhancement, will look into this. As always PRs are always welcome.

**Comment by BabyDino** [comment]
+1. This would be very helpful for us as well!

**Comment by ComfortablyCoding** [comment]
This has been added by #41 and should be available in the next release.

**Reply by ComfortablyCoding** [reply]
This functionality has been added in the latest release (v1.2.0)
